index.html

```diff
@@ -85,7 +85,21 @@ <h3 class="title-subheader">Aug 23-26  //  Portland, OR </h3>
 		</div>
 	</section>
 
-<!-- ### SITE NAV ### -->
+<!-- ## TABLET & DESKTOP NAV ### -->
+  <section class="nav-section">
+    <nav class="middleHeader">
+        <ul>
+          <a href="#schedule"><li>Schedule</li></a>
```

Comment:
When you click the [schedule tab](https://writespeakcode2017-staging.herokuapp.com/index.html#schedule) the height of the nav is the pre-sticky bar height, but all [the other tabs](https://writespeakcode2017-staging.herokuapp.com/index.html#sponsors) have the sticky bar height

I think this might be a case for "ship it then fix it", unless it's a fast fix